packages/EasyApiPlatform/src/Bridge/Symfony/EasyApiPlatformSymfonyBundle.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyApiPlatform\Bridge\Symfony;
+
+use EonX\EasyApiPlatform\Bridge\Symfony\DependencyInjection\Compiler\EasyCoreCompilerPass;
+use EonX\EasyApiPlatform\Bridge\Symfony\DependencyInjection\EasyApiPlatformExtension;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+final class EasyApiPlatformSymfonyBundle extends Bundle
+{
+    public function build(ContainerBuilder $container): void
+    {
+        parent::build($container);
```

Comment:
No need for that as the parent method doesn't do anything.

Reply:
Done.